src/lib/apiTypesV2/Entity.cpp

```diff
@@ -103,9 +103,23 @@ std::string Entity::check(ConnectionInfo* ciP, RequestType requestType)
     return "No Entity ID";
   }
 
-  if (forbiddenChars(id.c_str()))         { return "Invalid characters in entity id";        }
-  if (forbiddenChars(type.c_str()))       { return "Invalid characters in entity type";      }
-  if (forbiddenChars(isPattern.c_str()))  { return "Invalid characters in entity isPattern"; }
+  if (forbiddenChars(id.c_str()))
```

Comment:
To avoid "double/triple/quadruple logging" you need a whole framework, to push errors on an "error-stack" and finally present the entire error-set on a high level. Each level gives its part of the knowledge of the error.
We don't have anything like that in the broker and to implement it, it is pretty costy.

I am sure we have lots and lots of 'double logs' in the code.

For this specific error we could simply remove the calls to LM_W on one of the two levels.
Just pick which level to keep.